.github/workflows/validate.yml

```diff
@@ -6,32 +6,26 @@ on:
     branches: [ master ]
 jobs:
   validate:
+    env:
+      GOLANG_CI_LINT: go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.43.0
     name: validate
     runs-on: ubuntu-latest
-    env:
-      GO111MODULE: on
     steps:
-      - name: Set up Go 1.13
+      - name: Set up Go 1.17
         uses: actions/setup-go@v2
         with:
-          go-version: 1.13
+          go-version: 1.17
         id: go
 
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: install
-        run: |
-          go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.20.0
-          go install github.com/golangci/golangci-lint/cmd/golangci-lint
-          golangci-lint --version
-
       - name: lint
         run: |
-          golangci-lint run --disable wsl ./...
+          ${GOLANG_CI_LINT} run --disable wsl ./...
           cd v2
           # It isn't allowed to shallow-copy protobuf, therefore govet is disabled until this is fixed in the entire repo
-          golangci-lint run --disable wsl,govet ./...
+          ${GOLANG_CI_LINT} run --disable wsl,govet ./...
```

Comment:
Perhaps use the action that they provide? 
```
- name: Run golangci-lint
  uses: golangci/golangci-lint-action@v2
  with:
      version: v1.43.0
      working-directory: v2
```
You might need to add a config file for disabling wsl and govet?